data/json/mapgen/abandoned01.json

```diff
@@ -570,5 +570,80 @@
         { "item": "remington_700", "x": 13, "y": 15, "chance": 80, "ammo": 100 }
       ]
     }
+  },
```

Comment:
Added an alternate barn without the suicide vignette, as it turns out it was heavily inflating shotgun counts.
Also having a suicide scene in every abandoned barn is a bit much.

Reply:
Might be better to make this overmap special fully unique since its got quite distinctive feautures, (I'd have sworn it already was unique tbh). Also you are missing the kids tomb and recently abandoned car in the field next to the barn, that and the set piece ought to spawn/not spawn together.

Reply:
Yea it's unique but not GLOBAL_UNIQUE, so that means it has a 30% chance of appearing per overmap.
Global unique would be fine, especially compared to my hack job here.